lib/Analizo/Batch/Job.pm

```diff
@@ -149,7 +149,7 @@ sub execute {
   # calculate metrics
   my $metrics_cache_key = "metrics://$tree_id";
   my $metrics = $self->cache->get($metrics_cache_key);
```

Comment:
I suppose make more sense avoid get the cache here when $nocache is true with something like:

`my $metrics = $self->cache->get($metrics_cache_key) unless $nocache`;